prow/cmd/tide/config.md

```diff
@@ -15,6 +15,12 @@ The following configuration fields are available:
 * `merge_method`: A key/value pair of an `org/repo` as the key and merge method to override
    the default method of merge as value. Valid options are `squash`, `rebase`, and `merge`.
    Defaults to `merge`.
+* `merge_approve_tag`: The tag appended to the automatic commit
+   message for each approver.  A recommended value (if you want this
```

Comment:
Do we really to allow people to customize the value of "Reviewed-by" rather than just determining whether the feature is enabled?

Reply:
> Do we really to allow people to customize the value of "Reviewed-by" rather than just determining whether the feature is enabled?

Homu decided to go with "Approved by" and Git and Linux both mean very specific things with `Reviewed-by` (see the links in my initial PR message; they're also in the commit message).  So I'd rather leave this up to the Tide configuration instead of hard-coding a string.  I'm willing to change to a hard-coded tag if that's a requirement to get this PR landed though ;).

Reply:
SGTM, don't feel strongly about this (despite commenting on it twice, heh)